exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,101 @@
+{
+   "book-store": {
+      "description": [
+                       "Calculate lowest price for shopping basket only ",
+                       "containing books from a single series"
+                     ],
+      "cases": [
+         {
+            "description": "A basket containing only a single book.",
+            "basket": [1],
+            "targetgrouping": [1],
```

Comment:
compare the type of `targetgrouping` in the below case: an array of arrays of integers: `[[2], [2]]`, versus this one here, an array of integers: `[1]`.

Should it be `[[1]]` for consistency?

Same question for all other cases where the type is array of integer rather than array of arrays of integers.